src/main/java/org/opensearch/knn/common/KNNConstants.java

```diff
@@ -109,8 +109,25 @@ public class KNNConstants {
     public static final String FAISS_SIGNED_BYTE_SQ = "SQ8_direct_signed";
     public static final String FAISS_SQ_CLIP = "clip";
 
-    // JVector specific constants
+    /***** JVector specific constants *****/
     public static final String JVECTOR_NAME = "jvector";
+    // Query related params
+    public static final String METHOD_PARAMETER_OVERQUERY_FACTOR = "overquery_factor";
+    public static final String METHOD_PARAMETER_THRESHOLD = "advanced.threshold";
+    public static final String METHOD_PARAMETER_RERANK_FLOOR = "advanced.rerank_floor";
+    public static final String METHOD_PARAMETER_USE_PRUNING = "advanced.use_pruning"; // TODO: wire this after jvector upgrade
+    public static final int DEFAULT_OVER_QUERY_FACTOR = 5; // We will query 5x more than topKFor reranking
+    public static final Double DEFAULT_QUERY_SIMILARITY_THRESHOLD = 0.0;
+    public static final Double DEFAULT_QUERY_RERANK_FLOOR = 0.0;
+    public static final Boolean DEFAULT_QUERY_USE_PRUNING = false; // TODO: wire this after jvector upgrade
+
+    // Construction related params
+    public static final String METHOD_PARAMETER_ALPHA = "advanced.alpha";
+    public static final String METHOD_PARAMETER_NEIGHBOR_OVERFLOW = "advanced.neighbor_overflow";
+    public static final String METHOD_PARAMETER_HEIRARCHY_ENABLED = "heirarchy_enabled"; // TODO: wire this after jvector upgrade
```

Comment:
There's a typo in the word hierarchy in the variable name and the string.

Reply:
that's embarrassing, fixed :)